lorien/Assets/I18n/fr.txt

```diff
@@ -0,0 +1,118 @@
+TOOLBAR_TOOLTIP_NEW_FILE        Créer un nouveau fichier (CTRL+N)
+TOOLBAR_TOOLTIP_OPEN_FILE       Ouvrir un fichier (CTRL+O)
+TOOLBAR_TOOLTIP_SAVE_FILE       Enregistrer un fichier (CTRL+S)
+TOOLBAR_TOOLTIP_UNDO            Annuler l'action (CTRL+Z)
+TOOLBAR_TOOLTIP_REDO            Rétablir l'action (CTRL+Y)
+TOOLBAR_TOOLTIP_BRUSH_TOOL      Pinceau (B)
+TOOLBAR_TOOLTIP_LINE_TOOL       Ligne (L)
+TOOLBAR_TOOLTIP_ERASER_TOOL     Gomme (E)
+TOOLBAR_TOOLTIP_PICKER_TOOL     Sélecteur de couleur (C)
+TOOLBAR_TOOLTIP_SELECT_TOOL     Outil de sélection (S)
+TOOLBAR_TOOLTIP_BRUSH_COLOR     Couleur de la brosse
+TOOLBAR_TOOLTIP_BRUSH_SIZE      Épaisseur de la brosse
+TOOLBAR_TOOLTIP_CANVAS_COLOR    Couleur du canevas
+TOOLBAR_TOOLTIP_GRID 			Afficher la grille
+
+# -----------------------------------------------------------------------------
+# Statusbar strings
+# -----------------------------------------------------------------------------
+
+STATUSBAR_POSITION              Position
+STATUSBAR_ZOOM                  Échelle
+STATUSBAR_PRESSURE              Pression
+STATUSBAR_FPS                   FPS
+STATUSBAR_STROKES               Courbes # TODO : check if "Tracés" could be a better fit
+STATUSBAR_POINTS                Points
+
+# -----------------------------------------------------------------------------
+# Settings strings
+# -----------------------------------------------------------------------------
+
+SETTINGS_TITLE                  Paramètres
+SETTINGS_GENERAL                Général
+SETTINGS_APPEARANCE             Apparence
+SETTINGS_RENDERING              Affichage
+SETTINGS_PRESSURE_SENSITIVITY   Sensibilité de la pression
+SETTINGS_BRUSH_SIZE             Épaisseur par défaut de la brosse
+SETTINGS_BRUSH_COLOR            Couleur par défaut de la brosse
+SETTINGS_CANVAS_COLOR           Couleur par défaut du canevas
+SETTINGS_PROJECT_FOLDER         Dossier par défaut du projet
+SETTINGS_LANGUAGE               Langue
+SETTINGS_THEME                  Thème
+SETTINGS_AA_METHOD              Méthode d'anticrénelage
+SETTINGS_AA_METHOD_NONE         Aucun
+SETTINGS_FPS_FOREGROUND         Fps lorsque actif
+SETTINGS_FPS_BACKGROUND         Fps lorsque inactif
+SETTINGS_BRUSH_CAPS             Extrémités des tracés
+SETTINGS_BRUSH_CAPS_FLAT        Plates
+SETTINGS_BRUSH_CAPS_ROUND       Arrondies
+SETTINGS_RESTART_NOTICE  		Un redémarrage est nécessaire pour appliquer les changements
+
+# -----------------------------------------------------------------------------
+# About dialog strings
+# -----------------------------------------------------------------------------
+
+ABOUT_DIALOG_TITLE              A propos
+ABOUT_DIALOG_COPYRIGHT          © 2021 Marcus Brummer (mbrlabs) et contributeurs 
+ABOUT_DIALOG_LICSENSE           Lorien est sous licence :
+ABOUT_DIALOG_BASED_ON           Lorien est basé sur :
+ABOUT_DIALOG_EASTEREGG          Caractère secret : # TODO : check the real meaning
```

Comment:
The word easteregg is probably best understood in in the original English. German for example has no translation as well. See french: https://fr.wikipedia.org/wiki/Easter_egg#:~:text=Un%20easter%20egg%20(terme%20anglais,de%20touches%20ou%20de%20clics.

Reply:
Yeah, names shouldn't be translated, I totally forgot that ! Also, French technical terms are rather unknown anyway. Thanks for the link, I agree, In French we don't have an exact translation for this word. 

Also, I think I misunderstood “Easter egg character” as I translated it like a “char” rather than a “person” (I forgot to check the link…).  I will change this to “Personnage Easter egg”, it's a better fit obviously, and it is commonly used withing French gaming websites.